extensions-core/histogram/src/main/java/org/apache/druid/query/aggregation/histogram/ApproximateHistogram.java

```diff
@@ -419,7 +419,7 @@ protected void mergeInsert(final int mergeAt, int insertAt, final float v, final
       // use unused slot to shift array left or right and make space for the new bin to insert
       if (insertAt < unusedIndex) {
         shiftRight(insertAt, unusedIndex);
-      } else if (insertAt >= unusedIndex) {
+      } else { // insertAt >= unusedIndex
```

Comment:
How about turning this into an `assert` instead of a comment (or just delete it, it's pretty obvious just from reading the code).

Reply:
Deleted